tests/test_examples.py

```diff
@@ -195,7 +195,8 @@ def test_mech_system_pendulum3(self):
         """
         This tests the class Mechanical_system together with ind3 and ida
         """
-        mech_system_pendulum.run_example('ind3',with_plots=False,with_test=True)
+        #Skip index3 as there seems to be an issue using Sundials 4.1

```

Comment:
Is it possible to run this example if the SUNDIALS version is OK? Otherwise with the change below we test nothing.

Reply:
I cannot see a way to get the sundials version from the Python api, only at the C level

Reply:
Hi @jschueller, sorry for the late reply I was away.

If the test is disabled moving forward then perhaps we should remove this test, @PeterMeisrimelModelon any ideas for parametrize this test? Do you think it will be possible with the addition of pytest? (#85)

Reply:
I wouldn't want to remove a test just because it doesn't work for a specific sundials version. Parametrization is an option, but still requires adding the sundials versions accessible from Python in some capacity, let me look into that.

Reply:
get_sundials_version() function incoming with https://github.com/modelon-community/Assimulo/pull/109, I'd suggest one can use that in an if inside that test and use a proper conditional skip once we get to pytest.

Reply:
works great, thanks, I cherry-picked it here and used it